### PR TITLE
Create a batch helper

### PR DESCRIPTION
**Pull request by double-beep**
This presents you a menu with everything. This file is ready, changes coming soon.
@helper-beep, also as a maintenair I must have more additions and deletions.

### CODE REVIEW

File: helper.bat [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+echo                        20 - Calculator
+echo                        21 - Paint
+echo                        22 - Chat With Other (Same Networked) Computers
+echo                        23 - DIRectory
+echo                        24 - ECHO ON
+echo                        25 - Character Map
+echo                        26 - Private Character Maker
+echo                        27 - Telnet
+echo                        28 - Batch Calculator
+
```
Comment (double-beep) [inline helper.bat]: Blanks, must be cleared

File: helper.bat [hunk 2]
```diff
@@ -0,0 +1,296 @@
+@echo off
+color 9
+Title Password Page
+:question
```
Comment (double-beep) [inline helper.bat]: Here we should put our name or register but **ONLY** username.
Then adding %user% or whatever.

Comment (double-beep) [reply helper.bat]: The color also is terrible.

File: helper.bat (hunk 1 above)
Comment (double-beep) [reply helper.bat]: All blanks of course

**Comment by double-beep** [comment]
You can review

**Comment by helper-beep** [comment]
Thanks.

**Comment by helper-beep** [comment]
Oops your reopen.

**Comment by double-beep** [comment]
So, now I have caught you!!! 👍 👍

**Comment by double-beep** [comment]
Merging.

**Pull request merged**